src/backend/distributed/shared_library_init.c

```diff
@@ -1465,7 +1465,7 @@ RegisterCitusConfigVariables(void)
 		&EnableVersionChecks,
 		true,
 		PGC_USERSET,
-		GUC_NO_SHOW_ALL,
+		GUC_NO_SHOW_ALL | GUC_NOT_IN_SAMPLE,
```

Comment:
should we define our own constant that encompasses both of these?

Reply:
Hmm, how about `GUC_HIDDEN`, or `GUC_NOT_FOR_GENERAL_USE` ?